css/stylez.css

```diff
@@ -0,0 +1,312 @@
+/*-__________ Top-level styles __________-*/
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	background: #556;
+	font-family: 'Open Sans', sans-serif;
+	color: #556;
+	font-size: .9em;
+}
+
+a {
+	text-decoration: none;
+}
+
+.wrapper {
+	padding-top: 80px;
+	min-height: calc(100vh - 46px);
+}
+
+article {
+	width: 30%;
+	float: left;
+	margin: 20px;
+	background: #f1f3ed;
+	border-radius: 10px;
+	height: 625px;
+	box-shadow: 0 4px 4px 6px rgba(0,0,0,.2);
+}
+
+.clearfix::after {
+	content: "";
+	display: table;
+	clear: both;
+}
+
+.strong {
+	font-weight: 600;
+}
+
+.main-footer {
+	background: #f2f2f2;
+	text-align: center;
+	width: 100%;
+	padding: 15px;
+	margin-top: 20px;
+}
+
+
+/*-__________ Header styles __________-*/
+
+.main-header {
+	background: #2589bd;
+	margin: 0;
+	padding: 0;
+	border-bottom: 3px solid #ace3b3;
+	position: fixed;
+	width: 100%;
+	box-shadow: 0 2px 2px rgba(0,0,0,.2);
+	z-index: 1000;
+}
+
+h1 {
+	margin: 20px;
+	font-weight: bold;
+	font-size: 2em;
+	color: #fff;
+	float: left;
+}
+
+.main-header a {
+	color: #fff;
+}
+
+.main-header a:hover {
+	color: #ace3b3;
+}
+
+.main-header a:hover .u_logo {
+	color: #f25f5c;
+}
+
+nav ul {
+	list-style-type: none;
+	font-size: 1.5em;
+	margin: 0;
+}
+
+nav li {
+	margin: 20px 50px;
+	float: right;
+}
+
+
+/*-__________ Class styles __________-*/
+
+/* General ====================*/
+
+.card_title {
+	margin: 0;
+	padding: 0;
+	text-align: center;
+	border-bottom: 1px solid #2b334f;
+	background: #f25f5c;
+	border-radius: 10px 10px 0 0;
+}
+
+h2 {
+	margin: 0;
+	padding: 10px;
+	font-size: 1.2em;
+}
+
+/* .product ====================*/
+
+/* Figure, image, caption */
+
+figure {
+	width: 320px;
+	height: 180px;
+	margin: 20px auto;
+	padding: 0;
+	position: relative;
+}
+
+.card_img {
+	width: 100%;
+	height: 100%;
+}
+
+figcaption {
+	background: rgba(37,137,189,.5);
+	color: #fff;
+	font-weight: 300;
+	position: absolute;
+	bottom: 0;
+	width: 100%;
+	padding: 5px;
+}
+
+figcaption p {
+	display: none;
+	font-size: .8em;
+	font-style: italic;
+}
+
+.fig_hover:hover {
+	height: 35%;
+}
+
+.fig_hover:hover p {
+	display: block;
+}
+
+/* .product_[name] */
+
+.product_desc p {
+	margin: 10px;
+	text-align: justify;
+	font-size: .9em;
+}
+
+.product_avail_logo {
+	width: 100%;
+	background: #2589bd;
+	color: #fff;
+	text-align: center;
+	padding: 5px;
+	font-weight: 600;
+}
+
+/* .specs ==================== */
+
+h3 {
+	font-size: 1.1em;
+	margin: 10px 5px 5px;
+}
+
+.list {
+	margin: 2px 15px 2px;
+	width: 30%;
+	font-size: .9em;
+}
+
+.left {
+	float: left;
+}
+
+.right {
+	float: right;
+}
+
+.left p,
+.right p {
+	display: block;
```

Comment:
Be careful about reiterating styles that are already set. `<p>` tags are already `display: block`